.github/workflows/semantic-release.yaml

```diff
@@ -1,39 +1,67 @@
-name: Semantic Release
-
-on:
-  push:
-    branches:    
-      - master
-
-jobs:
-  release:
-    runs-on: ubuntu-latest
-    steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
-        persist-credentials: true
-
-    - uses: actions/setup-node@v2
-      with:
-        node-version: '14'
-
-    - run: yarn install --frozen-lockfile
-
-    - name: Import GPG key
-      uses: crazy-max/ghaction-import-gpg@v3.0.1
-      with:
-        gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
-        passphrase: ${{ secrets.GPG_PASSPHRASE }}
-        git-user-signingkey: true
-        git-commit-gpgsign: true
-
-    - name: Build
-      run: yarn run build
-
-    - run: npx semantic-release
-      env:
-        GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
-        NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-        GIT_COMMITTER_NAME: github-actions[bot]
-        GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
\ No newline at end of file
+name: Semantic Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Build and release library
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: true
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+
+      - run: yarn install --frozen-lockfile
+
+      - name: Build
+        run: yarn run build
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v3.0.1
+        with:
+          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
+          passphrase: ${{ secrets.GPG_PASSPHRASE }}
+          git-user-signingkey: true
+          git-commit-gpgsign: true
+
+      - run: npx semantic-release
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADMIN_ACCESS_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GIT_COMMITTER_NAME: github-actions[bot]
+          GIT_COMMITTER_EMAIL: ${{ secrets.GIT_COMMITTER_EMAIL }}
+
+
+  update-tokens:
+    name: Update public JSONBin
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          persist-credentials: true
+
+      - id: tokensData
+        run: |
+          content=`cat ./data/tokens.json`
+          content="${content//'%'/'%25'}"
+          content="${content//$'\n'/'%0A'}"
+          content="${content//$'\r'/'%0D'}"
+          echo "::set-output name=data::$content"

```

Comment:
Makes sense, but can you add a comment explaining what each step is replacing in plain language please?

Reply:
I copied this piece of code to be honest ;). All I know for now, is that it allows to read multi line json.
I'll find out and add a comments.